src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -415,10 +415,10 @@ void InitExteriorConvectionCoeff(EnergyPlusData &state,
             TSky = GetCurrentScheduleValue(state, state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).SkyTempSchNum) +
                    DataGlobalConstants::KelvinConv;
         }
-        if (state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).GroundTempSchNum != 0) {
-            TGround = GetCurrentScheduleValue(state, state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).GroundTempSchNum) +
-                      DataGlobalConstants::KelvinConv;
-        }
+    }
+    if (state.dataSurface->UseSurfPropertyGndSurfTemp(SurfNum)) {
+        int gndSurfsNum = state.dataSurface->GroundSurfsPropertyNum(SurfNum);
+        TGround = state.dataSurface->GroundSurfsProperty(gndSurfsNum).SurfsTempAvg + DataGlobalConstants::KelvinConv;
     }
```

Comment:
Sets the ground surface temperature for LWR calculation if ground surface object is specified for an exterior surface.